docs/eventing/sources/apiserversource.md

```diff
@@ -0,0 +1,267 @@
+---
+title: "APIServerSource"
+linkTitle: "APIServerSource"
+weight: 31
+type: "docs"
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta1-red?style=flat-square)
+
+An APIServerSource brings Kubernetes API server events into Knative.
+
+## Installation
+
+The APIServerSource source type is enabled by default when you install Knative Eventing.
+
+## Example
+
+This example shows how to create an APIServerSource that listens to Kubernetes Events and
+send CloudEvents to a Knative Service.
+
+### Creating a namespace
+
+Create a new namespace called `apiserversource-example` by entering the following
+command:
+
+```shell
+kubectl create namespace apiserversource-example
+```
+
+### Create a Broker
```

Comment:
I think there is a general agreement that we want to keep the example very simple and standalone, meaning you shouldn't have to install anything else other than the source (in this case APIServerSource). Note that Knative distributes the source with a default broker but that's not always the case.

@abrennan89 is that also your understanding?

Reply:
Thanks for the comment! This should be fixed. I removed Broker and Trigger, and make event-display as the sink directly.